### DISCUSSION TITLE
How to compare the likelihood in two models?

### DISCUSSION THREAD

**Discussion by YuzhouGong** [hahnlab/CAFE5#164] [Q&A] [answered]
Hello author,
I used cafe5 with both single lambda and multi-lambda commands, and there were two Base_results.txt files in my result folders. The first line of my Base_results.txt was like:

for single lambda model:
Model Base Final Likelihood (-lnL): 318029

for multi-lambda model:
Model Base Final Likelihood (-lnL): 308483

I am wondering which model fits better (for now just consider the value of the likelihood without thinking their difference in a statistical singnificance)? If I simply compare the numbers, it seemed single lambda model outcompete multi-lambda model (because 318029 > 308483), but there was a negative sign befoer InL in the bracket, then the InL of the single lambda model was smaller than the InL of the multi-lambda model (for -318029 < -308483), which indicated the multi-lambda model fits better.

Please tell me in my case, which model is better? Thanks in advance!

**Comment by hahnlab-user** [comment, marked as answer]
See discussion post  #155, but that difference will be very significant in favor of the multi-lambda model.
